pom.xml

```diff
@@ -177,6 +178,16 @@ limitations under the License.
                 <version>${org.slf4j.version}</version>
                 <scope>provided</scope>
             </dependency>
+            <dependency>
+                <groupId>org.eclipse.jetty</groupId>
+                <artifactId>jetty-server</artifactId>
+                <version>${jetty.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.eclipse.jetty</groupId>
+                <artifactId>jetty-servlet</artifactId>
```

Comment:
Yup I will add the same Jetty section that NiFi uses:

  (ASLv2) Jetty
    The following NOTICE information applies:
       Jetty Web Container
       Copyright 1995-2015 Mort Bay Consulting Pty Ltd.